source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,320 @@
+Bulk export data
+=================
+At this time, the export supports attributes of the objects listed below.   Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. Deleted objects are not yet supported as well.  For requests to add additional attributes or objects to our exporter, please add a feature request on our `feature idea formum <https://mattermost.uservoice.com/forums/306457-general>`_.  
+
+Version object
+--------------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">type</td>
+      <td valign="middle">string</td>
+      <td>Must be the string "version"</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">version</td>
+      <td valign="middle">number</td>
+      <td>Must be the number 1.</td>
+    </tr>
+  </table>
+  
+Team object
+-----------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">name</td>
+      <td valign="middle">string</td>
+      <td>The team name.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">display_name</td>
+      <td valign="middle">string</td>
+      <td>The display name for the team.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">type</td>
+      <td valign="middle">string</td>
+      <td>The type of team. Can have one the following values:<br>
+          <kbd>"O"</kbd> for an open team<br>
+          <kbd>"I"</kbd> for an invite-only team.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">description</td>
+      <td valign="middle">string</td>
+      <td>The team description.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">allow_open_invite</td>
+      <td valign="middle">bool</td>
+      <td>Whether to allow open invitations. Must have one of the following values:<br>
```

Comment:
And here. Also, is it really necessary to spell out the possible values of a boolean field?